packages/kbn-es/src/serverless_resources/security_roles.json

```diff
@@ -120,7 +120,12 @@
           "privileges": ["read", "write"]
         },
         {
-          "names": ["metrics-endpoint.metadata_current_*", ".fleet-agents*", ".fleet-actions*", "risk-score.risk-score-*"],
+          "names": [
+            "metrics-endpoint.metadata_current_*",
+            ".fleet-agents*", ".fleet-actions*",
+            "risk-score.risk-score-*",
+            ".entities.v1.latest.security_*"
```

Comment:
This is the only reference to the `risk-score.risk-score-*` index on this file. I don't know what it does, but it looks wrong to me.

Reply:
The last time I updated these files, these roles are supposed to be a copy of [roles file](https://github.com/elastic/project-controller/blob/main/internal/project/security/config/roles.yml) on the project-controller repo. Looks like those Kibana roles have been moved to https://github.com/elastic/elasticsearch-controller/blob/main/helm/values.yaml. 

Earlier, we used to create and merge a PR on `project-controller` repo before creating a role update PR on Kibana. Not sure if you'll need to do the same with this new `elasticsearch-controller` repo and update the relevant files.